### PR TITLE
Implement unhide appointments and unhide patients

### PR DESCRIPTION
**Pull request by Guanzhou03**
idENTify now supports unhide patients and appointments previously hidden.
Example: `unhide appts t/nose ear` unhides previously hidden appointments with nose or ear tags.

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/UnhideAppointmentsCommand.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+import java.util.Objects;
+import java.util.function.Predicate;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.model.Model;
+import seedu.address.model.person.Appointment;
+import seedu.address.model.person.HiddenPredicateSingleton;
+
+/**
+ * Hides all persons and their appointments in address book whose name contains any of the argument keywords.
```
Comment (WingKei67) [inline src/main/java/seedu/address/logic/commands/UnhideAppointmentsCommand.java:17]: Should it be unhides?

File: src/main/java/seedu/address/logic/commands/UnhidePatientsCommand.java [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+import java.util.function.Predicate;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.model.Model;
+import seedu.address.model.person.HiddenPredicateSingleton;
+import seedu.address.model.person.Person;
+import seedu.address.model.person.predicates.AppointmentOfFilteredPersonsPredicate;
+
+/**
+ * Hides all persons and their appointments in address book whose name contains any of the argument keywords.
```
Comment (WingKei67) [inline src/main/java/seedu/address/logic/commands/UnhidePatientsCommand.java:18]: Same as above.

File: src/main/java/seedu/address/logic/commands/UnhidePatientsCommand.java [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+                        model.getFilteredAppointmentList().size()));
+    }
+
+    @Override
+    public boolean equals(Object other) {
+        if (other == this) {
+            return true;
+        }
+
+        if (!(other instanceof HidePatientsCommand)) {
```
Comment (WingKei67) [inline src/main/java/seedu/address/logic/commands/UnhidePatientsCommand.java:61]: I guess it's UnhidePatientsCommand right?

**Review by WingKei67 — APPROVED** [review]

File: src/main/java/seedu/address/logic/commands/UnhidePatientsCommand.java (hunk 2 above)
Comment (Guanzhou03) [reply src/main/java/seedu/address/logic/commands/UnhidePatientsCommand.java]: Fixed it, thank you

File: src/main/java/seedu/address/logic/commands/UnhidePatientsCommand.java (hunk 3 above)
Comment (Guanzhou03) [reply src/main/java/seedu/address/logic/commands/UnhidePatientsCommand.java]: Fixed it, thank you

**Review by JovonLim — APPROVED** [review]
LGTM

**Pull request merged**